using htlatex with pgf does now work out, fix from
https://tex.stackexchange.com/questions/185349/error-using-pgfsysdriver-with-tex4ht-only-shows-up-with-texlive-2014-ok-with-t
---
 texmf-dist/tex/generic/pgf/systemlayer/pgfsys-tex4ht.def |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- texlive-base-2015.20150625.orig/texmf-dist/tex/generic/pgf/systemlayer/pgfsys-tex4ht.def
+++ texlive-base-2015.20150625/texmf-dist/tex/generic/pgf/systemlayer/pgfsys-tex4ht.def
@@ -86,7 +86,7 @@
 %     class="}{\%s}{-\%s}{x-x-\%d}{}{">}{</span>}
 %}
 
-\def\pgfsys@svg@newline{\Hnewline}
+\def\pgfsys@svg@newline{^^J}
 
 % we don't want tex4ht putting <p> and </p> tags automatically in our svg code and causing invalid xhtml in our svg pictures
 \newif\ifpgfsys@CssIncluded@\pgfsys@CssIncluded@false
